components/lib/utils/DomHandler.js

```diff
@@ -647,10 +647,8 @@ export default class DomHandler {
              * @param {HTMLElement} node - Element to add
              */
             const addScrollableParent = (node) => {
-                if (hideOverlaysOnDocumentScrolling) {
-                    // For document/body/html elements, add window instead
-                    scrollableParents.push(node.nodeName === 'BODY' || node.nodeName === 'HTML' || node.nodeType === 9 ? window : node);
-                }
+                // For document/body/html elements, add window instead
+                scrollableParents.push(node.nodeName === 'BODY' || node.nodeName === 'HTML' || node.nodeType === 9 ? window : node);
```

Comment:
This was intentional to only do it if hidedocumentonScrolling is set. Some users want the behavior and others don't depending on their accessibility needs for closing overlays.  So this is not the right fix